src/client/java/teammates/client/scripts/FixingITESMMissingAccountScript.java

```diff
@@ -0,0 +1,68 @@
+package teammates.client.scripts;
+
+import com.googlecode.objectify.Key;
+import com.googlecode.objectify.cmd.Query;
+
+import teammates.storage.entity.Account;
+import teammates.storage.entity.Instructor;
+
+/**
+ * Script to fix the missing account in ITESM.
+ */
+public class FixingITESMMissingAccountScript extends DataMigrationEntitiesBaseScript<Instructor> {
```

Comment:
This should not be committed to the repo once used?

Reply:
Yup. After you approval, Prof Damith can run the fixing script. After that, I'll remove the commit for this file.